Sound/Tidal/Pattern.hs

```diff
@@ -4,7 +4,7 @@
 module Sound.Tidal.Pattern where
 
 import Control.Applicative
--- import Data.Monoid
+import Data.Monoid (mappend, mempty, Monoid())
```

Comment:
`Monoid()` here is unecessary - the class is imported as soon as you import the module